documentation/modules/rn-2.3.adoc

```diff
@@ -64,10 +64,6 @@ During a warm migration from {rhv-short}, the status of the target VM might beco
 
 The QEMU guest agent is not installed on migrated VMs. Workaround: Install the QEMU guest agent with a post-migration hook. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2018062[*BZ#2018062*])
 
-.Importer pod log is unavailable after warm migration
-
-Retaining the `importer` pod for debug purposes causes warm migration to hang during the precopy stage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2016290[*BZ#2016290*])
-
 As a temporary workaround, the `importer` pod is removed at the end of the precopy stage so that the precopy succeeds. However, this means that the `importer` pod log is not retained after warm migration is complete. You can only view the `importer` pod log by using the `oc logs -f <cdi-importer_pod>` command during the precopy stage.
 
 This issue only affects the `importer` pod log and warm migration. Cold migration and the `virt-v2v` logs are not affected.
```

Comment:
@RichardHoch Those lines should be removed as well

Reply:
Done. My mistake.

Reply:
Done. My mistake.